recipe/0261-non-rectangular-commenting/manifest.json

```diff
@@ -70,18 +70,23 @@
           "id": "{{ id.path }}/annotation/p0002-svg",
           "type": "Annotation",
           "motivation": "painting",
-          "resource": {
+          "body": {
+          	"language": "de",
+          	"type": "TextualBody",
+          	"value": "Gansenliessel"
+          },
+          "target": {
           	"type": "SpecificResource",
-          	"full": {
-          		"id": "https://iiif.io/api/image/3.0/example/reference/918ecd18c2592080851777620de9bcb5-gottingen/full/max/0/default.jpg",
-          		"type": "image/jpeg"
+          	"source": {
+          		"id": "https://iiif.io/api/image/3.0/example/reference/918ecd18c2592080851777620de9bcb5-gottingen",
```

Comment:
I think the source should be a canvas like so:
```
"source": {
          "id": "{{ id.path }}/canvas/p1",
          "type": "Canvas"
}
```
Although the spec has the following example:

```
{
     "type": "SpecificResource",
     "source": "https://example.org/iiif/book1/canvas/p3",
      "selector": {
            "type": "FragmentSelector",
            "value": "xywh=0,0,750,300"
      }
}
```
in https://iiif.io/api/presentation/3.0/#54-range
so you could just use:

```
"source": "{{ id.path }}/canvas/p1"
```

Reply:
I'm getting a build error, `source sequence is illegal/malformed utf-8` when I have 
```
          "target": {
            "type": "SpecificResource",
            "source": "{{ id.path }}/canvas/p1"
          }
```